db.go

```diff
@@ -88,6 +88,7 @@ type database struct {
 type Options struct {
 	Path     string
 	Readonly bool
+	Repair   bool
```

Comment:
I think repairing should be the default options, so perhaps change this to `NoRepair`?

Reply:
I'm unsure, being destructive by default seems dangerous.

Reply:
fair 'nough